modules/accessing-notebook-servers-owned-by-other-users.adoc

```diff
@@ -0,0 +1,31 @@
+:_module-type: PROCEDURE
+
+[id='accessing-notebook-servers-owned-by-other-users_{context}']
+= Accessing notebook servers owned by other users
+
+[role='_abstract']
+Administrators can access notebook servers that are owned by other users to correct configuration errors or help a data scientist troubleshoot problems with their environment.
+
+.Prerequisites
+* You are part of the {openshift-platform} administrator group. See _Adding administrative users for {openshift-platform}_ for more information.
+ifdef::self-managed[]
+* You have launched the Jupyter application. See link:{rhodsdocshome}{default-format-url}/getting_started_with_red_hat_openshift_data_science_self-managed/creating-a-project-workbench_get-started#launching-jupyter-and-starting-a-notebook-server_get-started[Launching Jupyter and starting a notebook server].
+endif::[]
+ifndef::self-managed[]
+* You have launched the Jupyter application. See link:{rhodsdocshome}{default-format-url}/getting_started_with_red_hat_openshift_data_science/creating-a-project-workbench_get-started#launching-jupyter-and-starting-a-notebook-server_get-started[Launching Jupyter and starting a notebook server].
+endif::[]
```

Comment:
Do these prereqs also need to be conditionalized for upstream/downstream? The module does exist upstream.

Reply:
yes fixed